engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbJobState.java

```diff
@@ -348,16 +349,28 @@ private void notifyJobAvailable(final DirectBuffer jobType) {
     }
   }
 
+  private void createJobRecord(final long key, final JobRecord record) {
```

Comment:
:question: Please also explain this change

Reply:
I've tried to explain this here: https://github.com/camunda-cloud/zeebe/pull/8802#discussion_r821897741